tools/docker/nats_image/Dockerfile

```diff
@@ -29,12 +29,16 @@ RUN apk add git
 
 RUN git clone --depth 1 https://github.com/nats-io/nats-server.git
 WORKDIR /src/nats-server
-RUN git checkout $NATS_VERSION
+RUN git fetch --tags && git checkout $NATS_VERSION
```

Comment:
This was needed to avoid the following error:
```
 => ERROR [linux/amd64 build  7/11] RUN git checkout v2.9.25                                                                                                                                        0.2s
 => ERROR [linux/amd64->arm64 build  7/11] RUN git checkout v2.9.25                                                                                                                                 0.2s
------
 > [linux/amd64 build  7/11] RUN git checkout v2.9.25:
#0 0.215 error: pathspec 'v2.9.25' did not match any file(s) known to git
------
------
 > [linux/amd64->arm64 build  7/11] RUN git checkout v2.9.25:
#0 0.214 error: pathspec 'v2.9.25' did not match any file(s) known to git
------
Dockerfile:32
--------------------
```